src/applications/coronavirus-chatbot/utils.js

```diff
@@ -17,12 +9,37 @@ const disableButtons = event => {
   }
 };
 
-export const watchForButtonClicks = () => {
+const disableCheckboxes = () => {
+  const inputs = document
+    .getElementById('webchat')
+    .getElementsByTagName('input');
+
+  for (let i = 0; i < inputs.length; i++) {
+    if (inputs[i].type === 'checkbox') {
+      inputs[i].disabled = true;
+    }
+  }
+};
```

Comment:
You could dodge the for-loop like - 


```suggestion
const disableCheckboxes = () => {
[...document.querySelectorAll('#webchat input[type="checkbox"]')]
    .forEach(input => input.disabled = true);
};
```